tests/Unit/Schema/Source/SchemaStitcherTest.php

```diff
@@ -17,7 +17,7 @@ public function setUp(): void
         parent::setUp();
 
         // @phpstan-ignore-next-line using the Safe variant crashes PHPStan
-        exec('mkdir --parents ' . self::SCHEMA_PATH);
+        exec('mkdir -p ' . self::SCHEMA_PATH);
```

Comment:
Please add a comment if you would like this change to stick, otherwise I am keen to refactor short CLI arguments into their long form when I see them 😉